lectures/inventory_ssd.md

```diff
@@ -338,9 +339,23 @@ v_init = jnp.zeros((K + 1, n_z), dtype=float)
 ```
 
 ```{code-cell} ipython3
-in_time = time.time()
+in_time0 = time()
 v_star, σ_star = solve_inventory_model(v_init, model)
-jax_time = time.time() - in_time
+jax_time0 = time() - in_time0
+print("Jax compilation plus execution time = ", jax_time0)
+```
+
+Let's run again to get rid of the compilation time.
+
+```{code-cell} ipython3
+in_time = time()
+v_star, σ_star = solve_inventory_model(v_init, model)
+jax_time = time() - in_time
+print("Jax execution time = ", jax_time)
+```
+
+```{code-cell} ipython3
+jax_time / jax_time0
```

Comment:
Thanks @mmcky . I reckon the `0` you mentioned above can be removed if without this line `jax_time / jax_time0` to compare the time for those two cases. 

Do you think we should keep or remove this line? If we keep it, then I suggest we modify it as
```
print("Jax compilation time = ", jax_time0 - jax_time)
```

Reply:
@shlff I think we should name these `jax_time_in` and `jax_time_out`

Reply:
Or something more explicit like `jax_time_with_compile` and `jax_time`

Reply:
Thanks @jstac and @mmcky . I agree with you that explicitness is better than implicitness.

Therefore maybe we adopt such a style
```
jax_time_in_with_compile = time()
xxx
jax_time_out_with_compile = time() - jax_time_in_with_compile
print("Jax compilation and execution time = ", jax_time_out_with_compile)

jax_time_in_without_compile = time()
xxx
jax_time_out_without_compile = time() - jax_time_in_without_compile
print("Jax execution time = ", jax_time_out_without_compile)
```
Under that style, there is no need to add a line like line 358 since it is explicit enough.

Give me a kudo if you think it is a good idea or feel free to leave your comments.

Reply:
Thanks @shlff , nice ideas.  

I actually think time "in" and time "out" might be confusing -- it's not so clear what in and out refer to.

I suggest

```
start = time()
xxx
jax_time_with_compile = time() - start
```

and so on.

Reply:
Thanks @jstac . That's even better, clear, simple, and explicit. 

Next, I will implement such changes if everyone is happy.

Reply:
Thanks @shlff , yes please.

Reply:
really nice suggestion @jstac